spec/factories/payments.rb

```diff
@@ -41,14 +41,4 @@
   trait :pending do
     status { Payment::PENDING }
   end
-
-  # This means that the HIPS status is 'expired', not the payment term status
-  trait :expired do
```

Comment:
**These should not be removed.**  They are still valid payment processor statuses.  They are defined as constants in Payment.

We need to have tests to ensure that we can handle (old) HIPS payments that still exist in the system.  Likewise with the /spec tests: Don't remove any tests that created / used  `:expired` or `:awaiting_payment` because we need to ensure that we can still deal with HIPS payments in that are in the system.